core/src/main/java/org/apache/accumulo/core/client/ScannerBase.java

```diff
@@ -341,6 +341,11 @@ default void fetchColumn(CharSequence colFam, CharSequence colQual) {
    * scan, only how quickly it is returned.
    *
    * <p>
+   * Using the hint {@code scan_type=<type>} and documenting all of the types for your application
+   * is one strategy to consider. This allows administrators to adjust executor and prioritizer
+   * config for your application scan types without having to change the application source code.
```

Comment:
You should also update the website (if needed): https://accumulo.apache.org/docs/2.x/administration/scan-executors

Reply:
apache/accumulo-website#157